api/src/wfl/service/firecloud.clj

```diff
@@ -126,6 +127,81 @@
                               {:Content/type "text/tab-separated-values"
                                :entities     (slurp file)})})))
 
+(defn import-entity-set
+  "
+  Formats existing sample ENTITIES into a Terra-compatible tsv
+  and uploads to WORKSPACE as an entity set entry identified by SET-NAME.
+
+  All entity references should correspond to existing entities.
+  All entity references should be of the same type.
+
+  Parameters
+  ----------
+    workspace - Fully-qualified Terra Workspace in which the entity set
+                will be created
+    set-name  - Unique identifier for the entity set
```

Comment:
I don't think this has to be unique right? I guess the entity will be updated if it already exists?

Reply:
Ah, the behavior when uploading with an existing identifier is that the members will be *appended* to the existing entity, and not overwritten as for `:entity` tsv uploads.  This felt undesirable to me and like something we'd wish to avoid.

https://github.com/broadinstitute/wfl/pull/353/files/5e8c027c361415cf372907d951a0eb9334e89a40#r609784639

When I ran the same entity set upload twice for the same id, the set size doubled from 2 -> 4 rather than being overwritten by the new upload.

Reply:
Suggest:
```clojure
(defn import-entity-set
  "Format sample ENTITIES into a TSV, and upload it to WORKSPACE as an
  entity set named SET-NAME.  WORKSPACE is a \"namespace/name\" string.
  ENTITIES must share the same type.

  (import-entity-set \"workspace-namespace/workspace-name\"
                     \"c25ee04b-...\"
                     [[\"sample\" \"NA12878\"] [\"sample\" \"NA12879\"]])"
  [workspace set-name [entity & _more :as entities]]
  (let [[type _name] entity]
    (->> entities
         (map (partial vector set-name))
         (util/columns-rows->terra-tsv :membership [type type])
         .getBytes
         (import-entities workspace))))
```